src/room/PCTransport.ts

```diff
@@ -56,12 +57,65 @@ export default class PCTransport extends EventEmitter {
   }
 
   async setRemoteDescription(sd: RTCSessionDescriptionInit): Promise<void> {
+    let mungedSDP: string | undefined = undefined;
     if (sd.type === 'offer') {
       let { stereoMids, nackMids } = extractStereoAndNackAudioFromOffer(sd);
       this.remoteStereoMids = stereoMids;
       this.remoteNackMids = nackMids;
+    } else if (sd.type === 'answer') {
+      const sdpParsed = parse(sd.sdp ?? '');
+      sdpParsed.media.forEach((media) => {
+        if (media.type === 'audio') {
+          // mung sdp for opus bitrate settings
+          this.trackBitrates.some((trackbr): boolean => {
+            if (!trackbr.transceiver || media.mid != trackbr.transceiver.mid) {
+              return false;
+            }
+
+            let codecPayload = 0;
+            media.rtp.some((rtp): boolean => {
+              if (rtp.codec.toUpperCase() === trackbr.codec.toUpperCase()) {
+                codecPayload = rtp.payload;
+                return true;
+              }
+              return false;
+            });
+
+            if (codecPayload === 0) {
+              return true;
+            }
+
+            let fmtpFound = false;
+            for (var fmtp of media.fmtp) {
```

Comment:
modern JS discourages using `var`

```suggestion
            for (const fmtp of media.fmtp) {
```